.github/CHANGELOG.md

```diff
@@ -406,6 +406,19 @@
 * Added the `id` attribute to templates, which was missing from 
   PR [(#1377)](https://github.com/PennyLaneAI/pennylane/pull/1377).
   [(#1438)](https://github.com/PennyLaneAI/pennylane/pull/1438)
+
+* The `qml.math` module, for autodiff framework-agnostic tensor manipulation,
```

Comment:
```suggestion
* The `qml.math` module for autodiff framework-agnostic tensor manipulation,
```

Reply:
Or do I sense an upcoming move away from the word "interface"? :)

Reply:
> Or do I sense an upcoming move away from the word "interface"? :)

😆 This is something I realized from the ADR. I'm trying to move towards a place where:

- `interface`: the specific code logic in PL for gluing together autodiff frameworks and quantum gradients
- `autodiff framework/library`: Torch, TF, etc.

Currently, `interface` means too many things, so I think it makes sense to narrow the scope.

Reply:
```suggestion
* The `qml.math` module, for framework-agnostic tensor manipulation,
```

Reply:
actually, feel free to leave the original then, it is more explicit!